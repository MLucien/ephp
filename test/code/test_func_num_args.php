<?php
function foo()
{
    $numargs = func_num_args();
    echo "Num args for <" . __FUNCTION__ . ">: $numargs\n";
}

foo(1, 2, 3);

