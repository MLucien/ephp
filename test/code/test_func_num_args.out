Num args for <foo>: 3
